src/main/drivers/max7456_symbols.h

```diff
@@ -25,6 +25,14 @@
 #define SYM_END_OF_FONT             0xFF
 #define SYM_BLANK                   0x20
 #define SYM_COLON                   0x2D
+#define SYM_BBLOG                   0x10
```

Comment:
One rule we have is that we generally only define symbols that are actually used - this makes it easy to sort out what slots are still available in a font.

So my recommendation is to comment out the defines for symbols that were added but aren't yet used, and uncomment them in the pull request that puts them to use.